internal/service/networkcontainer/resource_network_container_migration_test.go

```diff
@@ -57,13 +59,14 @@ func TestMigNetworkContainer_basicAzure(t *testing.T) {
 
 func TestMigNetworkContainer_basicGCP(t *testing.T) {
 	var (
-		projectID    = acc.ProjectIDExecution(t) // No mig.ProjectIDGlobal because network container
```

Comment:
[q] Now that migration and acceptance test run in the same package execution, is there value in preserving `mig.ProjectIDGlobal`? My understanding is that `ProjectIDExecution` could be used if the resource package is already using it for acceptance tests.

Reply:
you read my mind :-) i was planning a follow-up PR to delete mig.ProjectIDGlobal